src/plist.h

```diff
@@ -314,35 +314,54 @@ EXPORT_INLINE Obj NewImmutableEmptyPlist(void)
 /****************************************************************************
 **
 *F  NewPlistFromArray(<list>,<length>) . . create a plain list from a C array
+*F  NewPlistFromArgs(<args...>) .  create a plain list from list of arguments
+**
+**  This macro turns a variable-length list of macro arguments into an array,
```

Comment:
"this macro" when you are taking about two things? Also, one isn't a macro anymore? Or is it?

Reply:
Have (hopefully) clarified what's going on